packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -339,17 +347,29 @@ test(`zcf.makeInvitation - customDetails stratification`, async t => {
     description: 'whatever',
     installation: 'whatever',
     instance: 'whatever',
+    terms: {
+      brands: {},
+      issuers: {},
+    },
```

Comment:
Why are empty `brands` and `issuers` (and a duplicate of the data that will end up in `terms`) being included in the custom details? It seems a distracting and unenlightening choice to include in customTerms.

It could be anything, right? It doesn't have anything to do with the brand and issuers in the top-level terms. So it could completely contradict the real info according to Zoe. Wouldn't a better example either  show that these brands and issuers  are completely separate info or show that they're arbitrary fields according to the contract?

Reply:
The point is to add `terms` itself to the regular invitation details, the part according to zoe, not the `customDetails`, which is not clearly separated as the part according to the contract.

I don't understand the reference to `customTerms`. IIUC, `customTerms` is the part of `terms` aside from the `brands` and `issuers`. If we think of the customTerms rather than the terms as being the parameters of the contract, that could be an argument for only including the `customTerms`, not the full `terms`, in the invitationDetails.

>  It doesn't have anything to do with the brand and issuers in the top-level terms. So it could completely contradict the real info according to Zoe. 

This is the key issue. The entire purpose of this PR is to introduce only the `terms` according to Zoe. The PR is trash until it succeeds at that.

The relevant lines are

```js
const instanceRecord = state.instanceState.getInstanceRecord();
...
   terms: instanceRecord.terms,
```
Is `state.instanceState.getInstanceRecord().terms` the terms according to Zoe? In what ways can the contract influence the contents of this field?

Reply:
I'm confused by the following snippet. Why does `terms` appear twice in these invitation details? The deeper one seems to be at the discretion of the contract (it's under `customDetails`). 

```
  t.deepEqual(details, {
    description: 'myInvitation',
    handle: details.handle,
    installation,
    instance,
    terms: {
      brands: {},
      issuers: {},
    },
    customDetails: {
      description: 'whatever',
      installation: 'whatever',
      instance: 'whatever',
      terms: {
        brands: {},
        issuers: {},
      },
    },
  });
```